src/main/java/org/cascadebot/cascadebot/permissions/objects/ResultCause.java

```diff
@@ -0,0 +1,31 @@
+package org.cascadebot.cascadebot.permissions.objects;
+
+public enum ResultCause {
+
+    /**
+     * Indicates that this result originated from a permission in a group.
+     */
+    GROUP,
```

Comment:
we should maybe re-name these to something like DENY_GROUP in case we end up having a deny and allow needing the same name, and also organizes it better.